test/libsolidity/semanticTests/abiEncoderV2/cleanup/dynamic_array.sol

```diff
@@ -14,8 +14,8 @@ contract C {
     }
 }
 // ====
-// compileViaYul: also
 // EVMVersion: >homestead
+// compileViaYul: also
```

Comment:
This PR would be shorter and a bit less intimidating if you could remove changes that now don't add gas expectations but only move this line.

Reply:
I have implemented enforcing gas tests with some threshold value and used that to activate tests that have usage greater than 100000.

I think it should be fine now